### ISSUE TITLE
Multiple Mega accounts

### ISSUE THREAD

**Issue by yesyeahvh123** [schmwong/keep-accounts-active#11]
i have a large number of Mega accounts, how to add user/password without having to create multiple secret USR_MEGA_number and PWD_MEGA_number, can you create a loop and combine multiple user into 1 secret USR_MEGA and multiple passwords into 1 secret PWD_MEGA separate lines?

**Comment by schmwong** [comment]
Sounds good, I'm working on another branch to address this. Updating shortly.

**Comment by schmwong** [comment]
Do comment in this [pull request](https://github.com/schmwong/keep-accounts-active/pull/12) if the updated code works on your end 😃

**Comment by yesyeahvh123** [comment]
Thank you so much sir

**Comment by schmwong** [comment]
You will also need a **Personal Access Token** with `workflow` and `full control of private repo` permissions enabled. `$WORKFLOW_TOKEN` uses that in the yml files in order to access your private repo and to edit the cron schedule in your workflow file.

**Comment by yesyeahvh123** [comment]
sir Can you tell me what's wrong? [Here](https://github.com/yesyeahvh123/keep-accounts-active/runs/8219761096?check_suite_focus=true)

**Comment by yesyeahvh123** [comment]
can you check my repository Is there something wrong?

**Comment by yesyeahvh123** [comment]
do i have to create a file at  private repo login-log?

**Comment by yesyeahvh123** [comment]
![Capture](https://user-images.githubusercontent.com/69633089/188775818-55054a49-dfcc-4237-a5fd-539bf6f9391c.PNG)

**Comment by schmwong** [comment]
Just read your logs. 
```
Run git clone \
Cloning into 'login-log'...
warning: You appear to have cloned an empty repository.
```
Yes, you have to create a file in your mega folder in the private repo.

What I did was to create an empty .gitkeep file (you can name the file anything you want), 
so that the mega folder can be found (GitHub does not allow empty folders).

![Screen Shot 2022-09-07 at 11 17 09 AM](https://user-images.githubusercontent.com/79643071/188781184-c0a30c1c-4fdb-4151-8a15-b92e16a1ed65.png)


Sorry for the confusion, I should have included all this info in the readme documentation. (Didn't expect anyone else to use this script so early in development 😅)

**Comment by yesyeahvh123** [comment]
thank you sir it working now

**Comment by yesyeahvh123** [comment]
Sir can you create auto claim free game epicgames?

**Comment by schmwong** [comment]
Still clearing assignments now, I will need a few more days to start thinking about more browser automation.

**Comment by yesyeahvh123** [comment]
awesome, you really did it, thank you so much, Secrets is email and password similar to Mega right?

**Comment by schmwong** [comment]
Yes, secrets is in the same format. But the script can't run properly on GitHub Actions yet, it's stuck at the login step. Not sure if it's due to epic games detecting logging in from a different location.

So for now it just runs on the local machine as a scheduled task :/
